pyvera/__init__.py

```diff
@@ -645,3 +653,91 @@ def is_locked(self, refresh=False):
             self.refresh_complex_value('Status')
         val = self.get_complex_value('Status')
         return val == '1'
+
+class VeraThermostat(VeraDevice):
+    """Class to represent a thermostat."""
+    def get_payload(self, service_name, name, value):
+        return {
+            'id': 'lu_action',
+            'output_format': 'json',
+            'DeviceNum': self.device_id,
+            'serviceId': service_name,
+            'action': 'Set' + name,
+            'New' + name: value}
+
```

Comment:
this method is currently class in line 310, if the decedent class doesn't override this (every class except VeraThermostat) then the base class provides the value.